Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -135,30 +188,111 @@ void HiresTexture::Update()
       }
     }
 
-    s_textureCacheAbortLoading.Clear();
+    s_texturePrefetcher_AbortLoading.Clear();
     s_prefetcher = std::thread(Prefetch);
   }
 }
 
 void HiresTexture::Prefetch()
 {
   Common::SetCurrentThreadName("Prefetcher");
+  // bytes to MBytes calcs will auto convert stuff to double
+  // so why not use double for the rest and config so it's consistent with printf %.1f
+  // because this is still WIP - remove_this_when_done
+
+  double prefetch_starttime_total = 0;
+  double prefetch_stoptime_total = 0;
+
+  // Default 30% -- Todo GUI User Configurable, INI Only for now
+  double percent_mem_to_cull = 30.0;  // aka culled_percent
+
+  if (Config::Get(Config::GFX_HIRESTEX_OVERRIDE_PREFETCH_CULLED_PERCENT) != 30.0)
+  {
+    if ((Config::Get(Config::GFX_HIRESTEX_OVERRIDE_PREFETCH_CULLED_PERCENT) < 5.0) ||
+        (Config::Get(Config::GFX_HIRESTEX_OVERRIDE_PREFETCH_CULLED_PERCENT) > 90.0))
+    {
+      if (Config::Get(Config::GFX_HIRESTEX_OVERRIDE_PREFETCH_CULLED_PERCENT) < 5.0)
+      {
+        percent_mem_to_cull = 5.0;
+        OSD::AddMessage(
+            "Prefetcher: User INI setting for 'culled cercent' of " +
+                StringFromFormat(
+                    "%.1f %%", Config::Get(Config::GFX_HIRESTEX_OVERRIDE_PREFETCH_CULLED_PERCENT)) +
+                " is below the lowest bound. Resetting to lowest allowable value.",
+            20000);
+        WARN_LOG(VIDEO, "Prefetcher: Mem limit cull percent INI override out of "
+                        "lower bound. Setting to lowest");
+      }
+      else if (Config::Get(Config::GFX_HIRESTEX_OVERRIDE_PREFETCH_CULLED_PERCENT) > 90.0)
+      {
+        percent_mem_to_cull = 90.0;
```

Comment:
Assign 90.0, 30.0 and 5.0 to `constexpr` variables, then use those variables in the code instead of writing 90.0, 30.0 and 5.0 everywhere. That way, if someone wants to change those values in the future, they only have to do it in one place instead of several places. The code will also be easier to understand.